virtual_feature.pl

```diff
@@ -1441,6 +1441,8 @@ sub feature_save_web_php_directory
 		}
 	}
 
+&virtual_server::create_php_bin_links($d, $mode);
```

Comment:
What would happen if mode equals none .. ?

Reply:
Not sure, but I will add some code to `create_php_bin_links` to definitely do nothing if PHP is disabled..

Reply:
Alright, great. Thanks!